puma.gemspec

```diff
@@ -17,5 +17,6 @@ Gem::Specification.new do |s|
             %w[History.md LICENSE README.md]
   s.homepage = "http://puma.io"
   s.license = "BSD-3-Clause"
+  s.metadata["msys2_mingw_dependencies"] = "openssl"
```

Comment:
I think this would be better on line 16 underneath `s.extensions = ["ext/puma_http11/extconf.rb"]` seems to be more related to extensions than `homepage` & `metadata`

Reply:
Just changed it to your way. I wasn't 100% sure about where to put the line so this was helpful. Thanks for the tip!